modern/src/LoginPage.js

```diff
@@ -42,12 +40,10 @@ const useStyles = makeStyles(theme => ({
   },
 }));
 
-const LoginPage = () => {
+const LoginPage = ({ children }) => {
```

Comment:
It probably makes sense to rename `LoginPage` to something else. Maybe `StartPage` or `OnboardingPage` or something like that.